specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/AppConfiguration/tspconfig.yaml

```diff
@@ -22,6 +22,10 @@ options:
     generate-test: true
     generate-sample: true
     flavor: azure
+  "@azure-tools/typespec-rust":
+    crate-name: "azure_management_appconfiguration"
```

Comment:
Decision for CP was `azure_resourcemanager_` as a prefix: https://azure.github.io/azure-sdk/rust_introduction.html#rust-packaging-prefix